docs/starting-a-new-app.md

```diff
@@ -58,7 +58,7 @@ Included within the project template are a number of npm tasks available, with e
 * `npm run pack-p` - Packages the app into **./dist** in production mode (minified code, with development code dropped).
 * `npm run watch` - Packages in development mode and sets up a watcher that will rebuild the app whenever the source code changes.
 * `npm run test` - Builds and executes any test spec files within the project.
-* `npm run lint `- Lints the project's JavaScript files according to the Enact ESLint configuration settings.
+* `npm run lint `- Lints the project's JavaScript files according to the Enact ESLint configuration settings and optionally TSLint.
```

Comment:
Should we say 'and/or' here and above?

Reply:
I think the wording here makes sense in that regardless javascript is checked (and using our Enact baked-in config) with TSLint being opt-in.  If you have something you think sounds clearer I'm open to suggestions, but I think this change is apt.

Reply:
OK.